packages/yoastseo/src/languageProcessing/languages/de/helpers/internal/determineStem.js

```diff
@@ -108,14 +106,21 @@ const findStemOnVerbExceptionList = function( morphologyDataVerbs, stemmedWord )
  * @returns {string} Stemmed form of the word.
  */
 export default function determineStem( word, morphologyDataGerman ) {
+	// Already return the stem here if the word contains umlaut and ends with an ending that looks like a valid suffix, e.g. "läden" stemmed to "laden".
+	const umlautException = morphologyDataGerman.nouns.umlautException;
```

Comment:
To avoid backward compatibility issue with older version of morphology data (like this issue [24.0: New Free + Old Premium: TypeError: Cannot read properties of undefined (reading 'list')-Assessments are broken](https://github.com/Yoast/plugins-automated-testing/issues/1993)), it's better to have a defensive coding here before accessing `umlautException`.
```suggestion
	const umlautException = morphologyDataGerman.nouns.umlautException || [];
```